templates/documentation/delivery-analytics/private-video-session-tokens.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: "Private Video Session Tokens"
+title: Private Video Session Tokens
+slug: private-video-session-tokens
```

Comment:
`slug` can be removed, as confirmed by Doctave [here](https://api-video.slack.com/archives/C05DTNR3QDS/p1697618392282579)

Reply:
Done ✅